platformio.ini

```diff
@@ -12,7 +12,7 @@
 default_envs = rack32-debug
 
 [firmware]
-name = \"OXRS-SHA-StateMonitor-ESP32-FW\"
+name = \"OXRS-SHA-StateMonitor-FW\"
```

Comment:
Should this be `OXRS-SHA-StateMonitor-ESP-FW`?

Reply:
I think we agreed on this 
![grafik](https://github.com/SuperHouse/OXRS-SHA-StateMonitor-ESP32-FW/assets/53935853/b81d9552-3be5-4b12-86b2-cc34ec375b0b)
see our discussion on Discord May, 3rd

Reply:
Apologies, yes I do recall now. As you were!